client/resource_group/controller/controller.go

```diff
@@ -88,20 +95,7 @@ type ResourceGroupsController struct {
 	tokenBucketUpdateChan chan *groupCostController
 
 	run struct {
```

Comment:
May I suggest removing the redundant field as it appears to only contain a single value?